crates/types/src/config/admin.rs

```diff
@@ -100,7 +100,7 @@ impl Default for AdminOptions {
             query_engine: Default::default(),
             heartbeat_interval: Duration::from_millis(1500).into(),
             // try to trim the log every hour
-            log_trim_interval: Some(Duration::from_secs(60 * 60).into()),
+            log_trim_interval: None,
```

Comment:
Is this for the demo? I'm not sure if we'll be using another build, but we can also increase this window to a 24h or something. wdyt?

Reply:
Yeah, this was meant for the demo. Setting it to 24h should be fine as well.